IBPSA/Fluid/Chillers/ModularReversible/Data/TableData2D/EN14511/Vitocal251A08.mo

```diff
@@ -0,0 +1,46 @@
+within IBPSA.Fluid.Chillers.ModularReversible.Data.TableData2D.EN14511;
+record Vitocal251A08 "A2W Vitocal 251 by Viessmann"
+  extends IBPSA.Fluid.Chillers.ModularReversible.Data.TableData2D.Generic(
+    tabLowBou=[283.15,280.15; 318.15,280.15],
+    dpEva_nominal=0,
```

Comment:
Enter `dpEva_nominal` and `dpCon_nominal` from the manufacturer data sheet.
If not available, set them to zero in the example; otherwise users will simulate models with these data assuming they capture the pressure drop.

Reply:
Same as above